omod/src/main/webapp/pages/results.gsp

```diff
@@ -1,29 +1,34 @@
-<%
-    ui.decorateWith("appui", "standardEmrPage")
-    ui.includeJavascript("uicommons", "datatables/jquery.dataTables.min.js")
-%>
-
+<% ui.decorateWith("appui", "standardEmrPage") %>
+<style>
+.ui-autocomplete {
+    max-height: 100px;
+    overflow-y: auto;
+    overflow-x: hidden;
+}
+</style>
+<script src="//cdn.datatables.net/1.10.15/js/jquery.dataTables.min.js"></script>
 <script type="text/javascript">
+    var patients = [];
     var breadcrumbs = [
         {icon: "icon-home", link: '/' + OPENMRS_CONTEXT_PATH + '/index.htm'},
         {
             label: "${ ui.message('dataintegrity.rule.label')}",
             link: '${ui.pageLink("dataintegrity", "rules")}'
         },
-        <% if(rule!= null) { %>
+        <% if(rules!= null) { %>
```

Comment:
@shivtej1505 This will break since you have a list here yet the expectation is to have only one